resource/opentelemetry-resource-detector-azure-app-service/tests/test_app_service_resource_detector.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import unittest
+from unittest.mock import patch
+
+from opentelemetry.resource.detector.azure.app_service import (
+    AzureAppServiceResourceDetector,
+)
+
+TEST_WEBSITE_SITE_NAME = "TEST_WEBSITE_SITE_NAME"
+TEST_REGION_NAME = "TEST_REGION_NAME"
+TEST_WEBSITE_SLOT_NAME = "TEST_WEBSITE_SLOT_NAME"
+TEST_WEBSITE_HOSTNAME = "TEST_WEBSITE_HOSTNAME"
+TEST_WEBSITE_INSTANCE_ID = "TEST_WEBSITE_INSTANCE_ID"
+TEST_WEBSITE_HOME_STAMPNAME = "TEST_WEBSITE_HOME_STAMPNAME"
+
+TEST_WEBSITE_RESOURCE_GROUP = "TEST_WEBSITE_RESOURCE_GROUP"
+TEST_WEBSITE_OWNER_NAME = "TEST_WEBSITE_OWNER_NAME"
+
+class TestAzureAppServiceResourceDetector(unittest.TestCase):
+    @patch.dict("os.environ", {
+        "WEBSITE_SITE_NAME": TEST_WEBSITE_SITE_NAME,
+        "REGION_NAME": TEST_REGION_NAME,
+        "WEBSITE_SLOT_NAME": TEST_WEBSITE_SLOT_NAME,
+        "WEBSITE_HOSTNAME": TEST_WEBSITE_HOSTNAME,
+        "WEBSITE_INSTANCE_ID": TEST_WEBSITE_INSTANCE_ID,
+        "WEBSITE_HOME_STAMPNAME": TEST_WEBSITE_HOME_STAMPNAME,
+        "WEBSITE_RESOURCE_GROUP": TEST_WEBSITE_RESOURCE_GROUP,
+        "WEBSITE_OWNER_NAME": TEST_WEBSITE_OWNER_NAME,
+    }, clear=True)
+    def test_on_app_service(self):
+        resource = AzureAppServiceResourceDetector().detect()
+        attributes = resource.attributes
+        # self.assertEqual(attributes {
```

Comment:
Should this commented out code be removed or uncommented?

Reply:
Removed